changelog/1465.trivial.1.rst

```diff
@@ -0,0 +1,3 @@
+An attempt to import PlasmaPy from a version of Python below the
```

Comment:
```suggestion
An attempt to import `plasmapy` from a version of Python below the
```

This a pkg reference instead of a project reference.  Importing the PlasmaPy project doesn't make sense.

Reply:
This also doesn't also doesn't satisfy the tense outlined in the change log guide.  Maybe something more like...

```rst
Changed the raised exception to `ImportError` (from a general `Exception`)
when attempting to import `plasmapy` from a Python version below the
minimum supported version.
```